cognite/extractorutils/uploader.py

```diff
@@ -560,3 +560,264 @@ def __len__(self) -> int:
             Number of events in queue
         """
         return self.upload_queue_size
+
+
+class SequenceUploadQueue(AbstractUploadQueue):
+
+    def __init__(
```

Comment:
The `TimeSeriesUploadQueue` has a `create_missing` option to enable the automatic creation of missing time series. For consistency, I suggest adding that as an option here as well, and leave it off by default.